content/docs/tasks/telemetry/servicegraph/index.md

```diff
@@ -1,10 +1,13 @@
 ---
-title: Generating a Service Graph
+title: Generating a Service Graph*
 description: This task shows you how to generate a graph of services within an Istio mesh.
 weight: 50
 keywords: [telemetry,visualization]
 ---
 
+> \* Note: The Service Graph has been deprecated.
```

Comment:
I'd recommend a warning icon here:

https://preliminary.istio.io/about/contribute/writing-a-new-topic/#adding-icons-emojis

`{{ warning }} The Service graph has been deprecated and replaced ...`